### DISCUSSION TITLE
fastlane match with signed commits

### DISCUSSION THREAD

**Discussion by hsheresd** [fastlane/fastlane#20981] [General]
Our organization uses signed commits. fastlane match works correctly when git config --global has been used to setup with the GPG settings, but if those GPG settings have only been setup in the local working directory, match fails with an error saying the commit was not signed. Is there any way to get around this without using git config --global?

**Comment by mahipalsingh-syt** [comment]
any update how to signed commit in fastlane?

**Comment by cupcakearmy** [comment]
Having the same issue here
